### DISCUSSION TITLE
Video Resizing for a DVD movie

### DISCUSSION THREAD

**Discussion by WISMD** [community/community#21435] [Other Feature Feedback, Questions, & Ideas]
<p>I am trying to resize The Stand.  The DVD is standard TV dimensions, but I can “zoom” with my TV and the video looks great.  however when I try to resize the image using the application, I can change one dimension but it always defaults back to the original.  How can I force the application to change the dimensions of the video?  Thank you.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/wismd">@WISMD</a> and welcome to the community.</p>
<p>The GitHub Support Community is for support of GitHub and its related products and, more generally, for discussion of software development topics. I’m not entirely sure what application you’re trying to find support for here, so I can’t take a guess to point you in the right direction for getting support for it.</p>
<p>I hope you’re able to find your answers.</p>

**Comment by Terroset** [comment]
<p>Hmm, the task is not an easy one if you want to resize your dvd video. If I were you I would first convert it and then resize it with the simplest video editor, even an online one so that you don’t need to download an entire software on your pc. I can suggest you one in case you haven’t resized it yet - <a href="https://www.fastreel.com/video-maker.html" data-bbcode="true" rel="noopener nofollow ugc">https://www.fastreel.com/video-maker.html</a></p>
